public/js/script.js

```diff
@@ -0,0 +1,8 @@
+
+function refresh() {
+    window.location.reload();
+}
+
+function goBack() {
+    window.history.back();
+  }
```

Comment:
thiếu eof và thừa dấu cách
```suggestion
}

```

Reply:
resolved